tests/e2e_tests/test_submit.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from playwright.sync_api import Page, expect
+
+from tests.e2e_tests.helpers import (
+    create_user_with_roles,
+    generate_email_address,
+    has_sent_la_confirmation_email,
+    login_via_magic_link,
+)
+from tests.e2e_tests.pages.submit import SubmitReportPage
+
+pytestmark = pytest.mark.e2e
+
+
+def test_submit_report(domains, authenticator_fund_config, page: Page):
+    email_address = generate_email_address(
+        test_name="test_submit_report",
+        email_domain="communities.gov.uk",
+    )
+
+    create_user_with_roles(
+        email_address,
+        ["PF_MONITORING_RETURN_SUBMITTER", "TF_MONITORING_RETURN_SUBMITTER"],
+    )
+
+    login_via_magic_link(
+        page,
+        email_address=email_address,
+        domains=domains,
+        fund_config=authenticator_fund_config,
+    )
+
+    request_data_page = SubmitReportPage(page, domain=domains.submit)
```

Comment:
the name of this variable feels a bit misleading?

Reply:
Fixed.